framework/doc/content/source/fvkernels/FVBoundedValueConstraint.md

```diff
@@ -0,0 +1,32 @@
+# FVBoundedValueConstraint
```

Comment:
This name to me implies boundedness everywhere, so I think maybe something like `FVPointBoundedValueConstraint` is better. Then it seems like we are introducing a canonical naming scheme where classes are either `FVPoint` or `FVIntegral`. (This isn't relevant to your code but relevant to the names here ... we have a `Bounds` system that applies everywhere and that is used in conjunction with PETSc's variational inequality system)

Reply:
Now that I've gone and looked at the source, I see that this is looking at more than just one point, so I'd adjust the following documentation. This is a very interesting approach that I've never seen before. You are applying a spatially *constant* source/sink such that the primal field never violates the bounds. The approach I am used to is applying a spatially *varying* source/sink (e.g. a *field* Lagrange multiplier) such that the primal field never violates the bounds. Since neither is physical, I suppose I'm fine with this approach I'm not familiar with. @fdkong what do you think about this? Have you ever seen this before?

Reply:
However, I would guess that you will be introducing, in an integral sense, a much larger source/sink to the system than you would with a spatially varying LM field